ci_opam.ml

```diff
@@ -118,7 +118,12 @@ let install ?(depopts="") ?(tests=false) args =
 let install_with_depopts depopts =
   install ~tests:tests_run ~depopts ["-v"];
   ?|~ "opam remove %s -v" pkg;
-  ?|~ "opam remove -a %s" (filter_base depopts)
+  ?|~ "opam remove -a %s" (filter_base depopts);
+  if tests_run then begin
+    install ~tests:false ~depopts ["-v"];
+    ?|~ "opam remove %s -v" pkg;
+    ?|~ "opam remove -a %s" (filter_base depopts);
+  end
```

Comment:
The idea is that dependencies may be specified wrongly in opam, e.g. a dependency is specified as test-only, whereas some .ml file would need it during a regular build. Without this commit Travis would only do a test with installing every dep, succeed, then a user  tries to install it and it breaks. With this commit it tests both scenarios (install test-only deps+run-test, install just build deps + just build)